File: eBibliotekaClient/src/app/data/interfaces/Payment.ts
export class PaymentListVM {
  id: number;
  reason: string;
  amount: number;
  paymentMethod: string;
  timeOfPayment: Date;

  /**
   *
   */
  constructor() {
    this.id = -1;
    this.reason = 'Učitavanje...';
    this.amount = -1;
    this.paymentMethod = 'Učitavanje...';
    this.timeOfPayment = new Date();
  }
}

File: eBibliotekaServer/MembershipModule/Models/Payment.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace eBibliotekaServer.MembershipModule.Models
{
    public class Payment
    {
        public int ID { get; set; }
        public string Reason { get; set; }
        public decimal Amount { get; set; }
        public string PaymentMethod { get; set; }
        public DateTime TimeOfPayment { get; set; }
        public int MembershipID { get; set; }

        [ForeignKey(nameof(MembershipID))]
        public Membership Membership { get; set; }
    }
}
